Enable emptyDir volumes by default

jsravn opened the pull request:
To make it easier to get started on development clusters and minishift.

Also changes flag to `usePersistentVolumes`.

marcoderama commented on enterprise-suite/values.yaml:
```diff
@@ -59,7 +59,7 @@ prometheusMemoryRequest: 250Mi
 # Set to true to use emptyDir for all volumes, disabling persistent volumes.
-useEmptyDirVolumes: false
+useEmptyDirVolumes: true
 # Default StorageClass to use for persistent volumes
```
Does the StorageClass comment need something along the lines of "ignored if `useEmptyDirVolumes` is `true`"?

jsravn commented:
Please re-review as I've changed the flag name.

jsravn commented:
@marcoderama yes I suppose it's possible. I was thinking we're still "release candidate" so it's okay to make breaking changes like this.

jsravn commented:
It's not particularly easy to preserve the old flag either (which would take precedence?).

jsravn commented:
Okay, I've changed it to preserve `useEmptyDirVolumes`. `usePersistentVolumes` takes precedence.

I also modified it so the PVCs are not deleted - which would delete the underlying storage - as a precaution.

jsravn commented:
@marcoderama I don't think you pushed the approve button? :)
